data/avro-schema/src/main/resources/avro/net/corda/data/ledger/utxo/contract.verification/VerifyContractsRequest.avsc

```diff
@@ -0,0 +1,39 @@
+{
+  "type": "record",
+  "name": "VerifyContractsRequest",
+  "doc": "Make a request for contracts verification of a ledger transaction.",
+  "namespace": "net.corda.ledger.utxo.contract.verification",
+  "fields": [
+    {
+      "name": "timestamp",
+      "type": {
+        "type": "long",
+        "logicalType": "timestamp-millis"
+      },
+      "doc": "Time ({@link java.time.Instant}) in milliseconds when the record was created."
+    },
+    {
+      "name": "holdingIdentity",
+      "type": "net.corda.data.identity.HoldingIdentity",
+      "doc": "The holding identity of the user making the request."
+    },
+    {
+      "name": "transaction",
+      "type": "bytes",
+      "doc": "The serialized transaction"
```

Comment:
This will be serialized `UtxoLedgerTransactionContainer` containing `WireTransaction`, `InputStateAndRefs` and `ReferenceStateAndRefs`:
https://github.com/corda/corda-runtime-os/blob/mbrkic-r3/CORE-9384/contract-verification-flow-processor/components/ledger/ledger-utxo-flow/src/main/kotlin/net/corda/ledger/utxo/flow/impl/verification/UtxoLedgerVerificationServiceImpl.kt#L46